frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -524,58 +519,54 @@ export const experimentLogic = kea<experimentLogicType>([
             () => [(_, props) => props.experimentId ?? 'new'],
             (experimentId): Experiment['id'] => experimentId,
         ],
-        experimentData: [(s) => [s.experiment], (experiment) => experiment],
         breadcrumbs: [
-            (s) => [s.experimentData, s.experimentId],
-            (experimentData, experimentId): Breadcrumb[] => [
+            (s) => [s.experiment, s.experimentId],
+            (experiment, experimentId): Breadcrumb[] => [
                 {
                     name: 'Experiments',
                     path: urls.experiments(),
                 },
                 {
-                    name: experimentData?.name || 'New',
+                    name: experiment?.name || 'New',
                     path: urls.experiment(experimentId || 'new'),
                 },
             ],
         ],
         variants: [
-            (s) => [s.experiment, s.experimentData],
-            (newExperimentData, experimentData): MultivariateFlagVariant[] => {
-                if (experimentData?.start_date) {
-                    return experimentData?.parameters?.feature_flag_variants || []
+            (s) => [s.experiment, s.experiment],
+            (newexperiment, experiment): MultivariateFlagVariant[] => {
```

Comment:
`newExperiment` and similarly below too

Reply:
D'oh, accidentally added this to the follow up instead: https://github.com/PostHog/posthog/pull/13343